.vscode/recommended-settings.json

```diff
@@ -3,5 +3,14 @@
   "editor.codeActionsOnSave": {
     "source.organizeImports": true
   },
-  "typescript.tsdk": "node_modules/typescript/lib"
+  "typescript.tsdk": "node_modules/typescript/lib",
+  "[scss]": {
```

Comment:
The CI has a validation check that all files have been run through this formatter.  VSCode will not format on save if there are multiple formatter options and no default set for a language.  Since running this specific formatter is essentially _required_ by the project, it makes sense to have them at least as suggested workspace configurations.